Formula/inxi.rb

```diff
@@ -0,0 +1,26 @@
+class Inxi < Formula
+  desc "Full featured CLI system information tool"
+  homepage "https://smxi.org/docs/inxi.htm"
+  url "https://github.com/smxi/inxi/archive/3.0.38-1.tar.gz"
+  version "3.0.38"
+  sha256 "5adbbe8145e970de1d516a404554e887806d31382de81d290e71fe270183c28a"
+
+  def install
+    bin.install("inxi")
+    man.install("inxi.1")
+
+    ["LICENSE.txt", "README.txt", "inxi.changelog"].each do |file|
+      prefix.install(file)
+    end
+  end
+
+  test do
+    assert `"#{bin}/inxi" | grep -- "#{version}"`
```

Comment:
This is a "more homebrew-y" approach, please use the same in the other tests. (the tests look good now though)
```suggestion
    assert_match version.to_str, shell_output("#{bin}/inxi")
```